data_collection/web/guides.rst

```diff
@@ -40,8 +40,47 @@ Example::
 
 .. todo:: Explain how to change page URL and title for virtual page views or link to SPA instructions page.
 
+User ID
+-------
+
+User ID is an additional session dimension in yours analytics data. When set you will be able to aggregate data by this dimension. You can use Tag Manger, default Piwik Pro tag to setup `User ID` without editing any JS code. To set it manually follow this guide.
```

Comment:
I will add how other products user it, yet I thought that such description is for higher level articles.

Reply:
If we have articles that explain it, we can only mention it here and link to them.

Reply:
I would refrain from using word `dimension` or `session dimension` as it may bring confusion with custom dimensions.

Another thing is about Tag Manager usage- this is hugely problematic cross-team issue. Overall as RevOps we don't advise to use User ID in Tag Manager, as customers are usually unable to configure it properly. They are not taking into account, that as effect of such a configuration, `setUserId` is invoked with each tracker execution. And when a situation comes that the User ID variable is not set (due to unavailability of element for example), user ID is cleared in such session. We already has such situations. My proposition for solving this was to change TM implementation but for now it's as it was.

Reply:
@RomanSek https://help.piwik.pro/support/getting-started/userid/

Reply:
- Changed dimension to parameter
- Added link to help center article
- Removed word `dimension`
- Removed reference to TagManager